online-store/store/src/main/java/com/issoft/auto/store/RandomStorePopulator.java

```diff
@@ -3,52 +3,27 @@
 import com.github.javafaker.Faker;
 import com.issoft.auto.domain.Category;
 import com.issoft.auto.domain.Product;
-import org.reflections.Reflections;
+import com.issoft.auto.domain.db.DBManager;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Set;
 
 public class RandomStorePopulator {
 
-    Reflections reflections = new Reflections("com.issoft.auto.domain.categories");
-    Set<Class<? extends Category>> namedSubCategories = reflections.getSubTypesOf(Category.class);
+    public List<Category> getCategoriesForShop() {
```

Comment:
you should create another populator, like DB populator, and do not touch RSP, and make it possible to simple switch/uncomment smthing and you another type of populator.